### PR TITLE
add "failed uploads" troubleshooting section 

### PR DESCRIPTION
**Pull request by fakela**
add "failed uploads" troubleshooting section  #4903

### CODE REVIEW

**Comment by fakela** [comment]
@pappasilenus @svekars Please review

**Comment by JoeyZwicker** [comment]
Thanks for the PR @fakela! We'll take a look a review asap. Should be pretty quick.

**Comment by fakela** [comment]
@JimmyWhitaker Thanks for the suggestions and the reviews

File: doc/docs/1.11.x/troubleshooting/general_troubleshooting.md [hunk 1]
```diff
@@ -94,3 +94,31 @@ signify the end of options and tell the shell to process the
...
+
+If it still gets oomkilled by k8s, there are a couple of environment variables you can set on your pachd deployment to limit the amount of memory that sidecar and pachd use.
+
+- ``STORAGE_UPLOAD_CONCURRENCY_LIMIT`` limits the parallelism to put files into the storage backend. Default is 100.
+- ``STORAGE_PUT_FILE_CONCURRENCY_LIMIT`` limits the number of parallel downloads pachd will initiate. Default is also 100.
+
+You may use a binary search technique to hone in on a value appropriate for a production pipeline:
+
+for ``cache_size``, max it out. If it works, halve it. If it oomkills, increase that value by 50%. and so on
+for the ``CONCURRENCY_LIMITS``, halve and increase by 50% until you get a value that works.
```
[the remark this answers is not in the document]
Comment (fakela) [reply doc/docs/1.11.x/troubleshooting/general_troubleshooting.md]: > I'm not sure if we need to explain binary search here, although I definitely think it's the right way to set these parameters. Perhaps reword to

I did make some minor changes based on your previous suggestions. What do you suggest I reword it to or should i remove the reference for binary search entirely

**Comment by JimmyWhitaker** [comment]
@fakela everything is looking good to me. Thanks again for the contribution! Would you mind also adding these changes to the master docs, 1.10 and 1.9 if it applies?

File: doc/docs/1.11.x/troubleshooting/general_troubleshooting.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JimmyWhitaker) [reply doc/docs/1.11.x/troubleshooting/general_troubleshooting.md]: Now that I'm looking at it again, I think it's all good. I had a hard time thinking of a better way to word it :)

**Comment by fakela** [comment]
@JimmyWhitaker I was a bit busy during the week I wanted adding it to the master docs later today.

Saw you already did that 🎉
Thanks alot.

I guess the PR looks good now

**Comment by JimmyWhitaker** [comment]
Yep! Realized it was probably fine to just add it to the master. Thanks for the contribution!!
